content/administration/upgrade.rst

```diff
@@ -1,264 +1,340 @@
-upgrade services you can expect.
+      .. important::
+         The database is unavailable throughout the process. If anything goes wrong, the platform
+         automatically reverts the upgrade, as it would be for a regular update. In case of success,
+         a backup of the database before the upgrade is created.
 
-Upgrade services covered by the SLA
------------------------------------
+      The update of your custom modules must be successful to complete the entire upgrade process.
+      Make sure the status of your staging upgrade is :guilabel:`successful` before trying it in
+      production. More information on how to upgrade your custom modules can be found in the
+      :ref:`upgrading customizations documentation <upgrade/upgrading_customizations>`.
 
-Databases hosted on Odoo's cloud platforms (Odoo Online and Odoo.sh) or self-hosted (On-Premise) can
-benefit from upgrade services at all times for:
+   .. group-tab:: On-premise
 
-- the upgrade of all **standard applications**;
-- the upgrade of all **customizations created with the Studio app**, as long as Studio is still
-  installed and the respective subscription is still active; and
-- the upgrade of all **developments and customizations covered by a maintenance of customizations
-  subscription**.
+      The command to upgrade a database to production is similar to the one of upgrading a test
+      database except for the argument `test`, which must be replaced by `production`:
 
-Upgrade services are limited to the technical conversion and adaptation of a database (standard
-modules and data) to make it compatible with the version targeted by the upgrade.
+      .. code-block:: console
 
-Upgrade services not covered by the SLA
----------------------------------------
+         $ python <(curl -s https://upgrade.odoo.com/upgrade) production -d <your db name> -t <target version>
 
-The following upgrade-related services are **not** included:
+      An upgraded production database can also be requested via the `Upgrade page
+      <https://upgrade.odoo.com/>`_.
 
-- the **cleaning** of pre-existing data and configurations while upgrading;
-- the upgrade of **custom modules created in-house or by third parties**, including Odoo partners;
-- lines of **code added to standard modules**, i.e., customizations created outside the Studio app,
-  code entered manually, and :ref:`automated actions using Python code
-  <studio/automated-actions/action>`; and
-- **training** on using the upgraded version's features and workflows.
+      .. important::
+         When requesting an upgraded database for production purposes, the copy is submitted without
+         a filestore. Therefore, the upgraded database filestore must be merged with the production
+         filestore before deploying the new version.
 
-.. note:: |assistance-contact|
+In case of an issue with your production database, you can request the assistance of Odoo via the
+`support page <https://odoo.com/help>`_.
```

Comment:
Same for the other one, when reading this paragraph, the only option that make sense for the user is to select "An issue related to my upgrade (production)", I think it would be redundant to have the image, as it is recommended that the documentation be kind of "minimalist"

However, I can pre-select the correct option with the URL, in this case with `https://odoo.com/help?stage=post_upgrade`, and `https://odoo.com/help?stage=migration` (feel free to try those URL to confirm it works haha)

What do you think ?

Reply:
I like the idea of putting the link with the preselected option, for both test and post upgrade issues :+1: 

N.B.: Even if something makes sense and seems logic, don't asume customers are gonna realise or follow them haha